Umbraco-Cloud/Upgrades/Moving-From-Courier-To-Deploy/index.md

```diff
@@ -10,47 +10,59 @@ Courier has been replaced by Deploy, and Deploy is highly specialized to be work
+
+Next, request a valid Deploy Key from the support heroes. Use the in-app chat in the Umbraco Cloud portal to get this.
 
 Prerequisites for using the guide is that you have a running site, that contains all metadata, like document types, data types and templates. 
 
 ## 1 Removing Umbraco Courier
 First you need to remove all Courier related files and folders.
 
+* /App_Plugins/Deploy
+* /bin/Archetype.Courier.dll
 * /bin/Concorde.CacheHandler.dll
 * /bin/Concorde.CacheHandler.LiveEdit.dll
 * /bin/Concorde.CacheHandler.UI.dll
 * /bin/Our.Umbraco.Courier.DataResolvers.dll
+* /bin/Umbraco.Courier.Contrib.Resolvers.dll
 * /bin/Umbraco.Courier.Core.dll
 * /bin/Umbraco.Courier.DataResolvers.dll
 * /bin/Umbraco.Courier.EventHandlers.V7.dll
+* /bin/Umbraco.Courier.FormsProvider.dll
 * /bin/Umbraco.Courier.Persistence.dll
 * /bin/Umbraco.Courier.Persistence.V7.NHibernate.dll
 * /bin/Umbraco.Courier.Providers.dll
 * /bin/Umbraco.Courier.RepositoryProviders.dll
-* /bin/Umbraco.Courier.Contrib.Resolvers.dll
-* /bin/Umbraco.Courier.FormsProvider.dll
-* /bin/Archetype.Courier.dll
-* /App_Plugins/Deploy
 * /config/courier.config
 * /config/splashes/noNodes.aspx
 * /umbraco/Plugins/Courier/Webservices/Repository.asmx
 
-Finally these 3rd party dlls has been part of Courier. If you are not using them in your custom code, you should ensure that they get deleted as well.
+Also, these 3rd party dlls have been part of Courier. If you are not using them in your custom code, you should ensure that they get deleted as well.
 
-* /bin/FluentNHibernate.dll
-* /bin/NHibernate.dll
-* /bin/lesi.Collections.dll
 * /bin/Antlr3.Runtime.dll
 * /bin/Castle.Core.dll
+* /bin/FluentNHibernate.dll
+* /bin/lesi.Collections.dll
+* /bin/NHibernate.dll
 
-Finally all `.courier` files found in `/data/Revision` needs to be deleted as well, as they will no longer be used. **NOTE:** Do not remove the `/data/Revision` folder itself, only the contents inside the `/data/Revision` folder - you will need this folder for Deploy as well.
+If you have a NuGet reference to "Umbraco.Courier.Contrib", you should uninstall that and delete the related .dll from `bin` as well.
+
+Finally all `.courier` files found in `/data/Revision` needs to be deleted as well, as they will no longer be used. 
+
+**NOTE:** Do not remove the `/data/Revision` folder itself, only the folders inside the `/data/Revision` folder - you will need this folder for Deploy as well.
 
 ## 2 Adding Umbraco Deploy
 
-Adding Umbraco Deploy will be simple, all content from the package received, except `/config/UmbracoDeploy.config` needs to be copied into the webroot.
+Adding Umbraco Deploy will be simple, all content from the ZIP received (except `/config/UmbracoDeploy.config`) needs to be copied into the webroot.
+
+**NOTE:** Do not overwrite the existing file `/config/UmbracoDeploy.config` with the empty one from the ZIP. It already has the correct settings for your site.
```

Comment:
```suggestion
:::note
Do not overwrite the existing `/config/UmbracoDeploy.config` file with the empty one from the ZIP. It already has the correct settings for your site.
:::
```

Reply:
Oh, I didn't know about the "note format" - cool :-)